SIL.WritingSystems.Tests/GlobalWritingSystemRepositoryTests.cs

```diff
@@ -301,5 +301,29 @@ public void Count_LdmlRemovedByAnotherRepo_ReturnsZero()
 				Assert.That(repo1.Count, Is.EqualTo(0));
 			}
 		}
+
+		[Test]
+		public void AllWritingSystems_LdmlCheckingSetEmptyandGetWSId()
+		{
+			using (var e = new TemporaryFolder("GlobalWritingSystemRepositoryTests"))
+			{
+				var repo1 = new GlobalWritingSystemRepository(e.Path);
+				var repo2 = new GlobalWritingSystemRepository(e.Path);
+
+				var ws = new WritingSystemDefinition("en-US");
+				repo1.Set(ws);
+				repo1.Save();
+				Assert.That(ws.WindowsLcid, Is.Empty);
+				// ensure that last modified timestamp changes
+				Thread.Sleep(1000);
+				ws = repo2.Get("en-US");
+				ws.WindowsLcid = "test";
```

Comment:
Please add a comment here:
"Make an arbitrary change to force a save of the ldml file."

Reply:
Added the commen in the GlobalWritingSystemRepositoryTest.cs file.